### ISSUE TITLE
Gem doesn't include executables

### ISSUE THREAD

**Issue by pixeltrix** [rails/dartsass-rails#1]
The gemspec doesn't include the platform specific executable files so you get this error:

```
% gem install dartsass-rails
Fetching dartsass-rails-0.1.0.gem
Successfully installed dartsass-rails-0.1.0
1 gem installed
% dartsass
+ ~/.gem/ruby/2.7.5/gems/dartsass-rails-0.1.0/exe/darwin/sass
Traceback (most recent call last):
	3: from ~/.gem/ruby/2.7.5/bin/dartsass:23:in `<main>'
	2: from ~/.gem/ruby/2.7.5/bin/dartsass:23:in `load'
	1: from /Users/andyw/.gem/ruby/2.7.5/gems/dartsass-rails-0.1.0/exe/dartsass:22:in `<top (required)>'
~/.gem/ruby/2.7.5/gems/dartsass-rails-0.1.0/exe/dartsass:22:in `exec': No such file or directory - ~/.gem/ruby/2.7.5/gems/dartsass-rails-0.1.0/exe/darwin/sass (Errno::ENOENT)
```

**Comment by dhh** [comment]
Doh. Fixed in 0.1.1.

**Comment by dmolesUC** [comment]
What's the rationale for embedding the executables instead of using [sass-embedded](https://github.com/ntkme/sass-embedded-host-ruby)?
